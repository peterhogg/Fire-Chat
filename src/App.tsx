import React, { Component } from 'react';
import './App.css';
import Chatbox from './components/chatbox';

class App extends Component {
  render() {
    return (
      <div>
	  	<Chatbox />
      </div>
    );
  }
}

export default App;
