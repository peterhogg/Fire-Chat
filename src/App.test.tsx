import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';


describe("Fire Chat", () =>{
	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<App />, div);
		ReactDOM.unmountComponentAtNode(div);
	});
	it('has a valid snapshot', () =>{
		const component = renderer.create(
			<App />
		);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

})
	
