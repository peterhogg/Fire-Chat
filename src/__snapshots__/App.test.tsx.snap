// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fire Chat has a valid snapshot 1`] = `
<div>
  <div
    className="chatbox"
  >
    <form
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        type="textbox"
        value=""
      />
      <button
        disabled={false}
        type="submit"
      >
        Send
      </button>
    </form>
  </div>
</div>
`;
